src/client/eppo-client.spec.ts

```diff
@@ -198,26 +198,28 @@ describe('EppoClient E2E test', () => {
     });
   });
 
-  describe('UFC General Test Cases', () => {
-    beforeAll(async () => {
-      global.fetch = jest.fn(() => {
-        return Promise.resolve({
-          ok: true,
-          status: 200,
-          json: () => Promise.resolve(readMockUFCResponse(MOCK_UFC_RESPONSE_FILE)),
-        });
-      }) as jest.Mock;
+  describe('UFC Shared Test Cases', () => {
+    const testCases = testCasesByFileName<IAssignmentTestCase>(ASSIGNMENT_TEST_DATA_DIR);
```

Comment:
No logic changes to this file--just reorganizing and use a map so test cases are named based on file name

![image](https://github.com/Eppo-exp/js-client-sdk-common/assets/417605/e6abc836-ba93-4e21-8f6a-7debdf21e763)